### PR TITLE
Remove mention of break and continue statements

### PR DESCRIPTION
**Pull request by MaybeJustJames**

### CODE REVIEW

File: 6_Loops.ipynb [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "\n",
     "## 6.1 Introduction\n",
     "\n",
-    "Another important feature of computer programs is that they can do the same thing over and over again with different information. To allow this, it is possible to use loops in the program; essentially a loop is executed until it runs out of data or the code decides to break out of it.\n",
+    "Another important feature of computer programs is that they can do the same thing over and over again with different information. To allow this, it is possible to use loops in the program; essentially a loop until a given condition is met.\n",
```
Comment (tmuylder) [inline 6_Loops.ipynb]: Another important feature of computer programs is that they can do the same thing over and over again with different information. To allow this, it is possible to use loops in the program; essentially a loop is executed until a given condition is met.

File: 6_Loops.ipynb [hunk 2]
```diff
@@ -268,27 +268,7 @@
...
-    "myNumbers = range(1,100)\n",
-    " \n",
-    "for myNumber in myNumbers:\n",
-    "    if myNumber == 5:   \n",
-    "        continue     # This means that the code within the for loop will be ignored if myNumber is equal to 5, we 'jump back' to the start and use the next number (6)\n",
-    "    print(myNumber)\n",
-    "\n",
-    "    if myNumber == 8:\n",
-    "        break        # This means we will exit the loop alltogether, all other values after this one will not be dealt with.\n"
+    "Here we now have two levels of code besides the main one; the **if** is checked for every value, but the print is only executed for numbers divisible by myDivider."
```
Comment (tmuylder) [inline 6_Loops.ipynb]: In the example above we have two levels of ...

**Review by tmuylder — COMMENTED** [review]
Grammatical changes

File: 6_Loops.ipynb (hunk 2 above)
Comment (MaybeJustJames) [inline 6_Loops.ipynb]:
```suggestion
    "Here we noe besides the main one; the **if** is checked for every value, but the print is only executed for numbers divisible by myDivider."
```

Comment (tmuylder) [inline 6_Loops.ipynb]:
```suggestion
    "In the example above, we have two levels of code besides the main one; the **if** is checked for every value, but the print is only executed for numbers divisible by myDivider."
```

File: 6_Loops.ipynb (hunk 1 above)
Comment (tmuylder) [inline 6_Loops.ipynb]:
```suggestion
    "Another important feature of computer programs is that they can do the same thing over and over again with different information. To allow this, it is possible to use loops in the program; essentially a loop is executed until a given condition is met.\n",
```

**Pull request merged**